admin-server/src/main/kotlin/org/stellar/admin/server/plugins/StaticFileRouting.kt

```diff
@@ -0,0 +1,13 @@
+package org.stellar.admin.server.plugins
+
+import io.ktor.server.application.*
+import io.ktor.server.http.content.*
+import io.ktor.server.routing.*
+
+/** Function to configure the static files routing of the application. */
+fun Application.configureStaticFilesRouting() {
+  routing {
+    // The UI files are served from the /ui path.
+    staticResources("/ui", "ui")
```

Comment:
I think UI should be a different server, but it should be a part of the docker image IMO

Reply:
I'm pretty sure it will make packaging easier. If we will have a react app, how are we going to ship it? Will UI be part of this repo? I'd personally prefer it to be in a separate repo

Reply:
After discussion in the standup, we decided to remove the static endpoint.